babel.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	presets: [
+		[
+			'@babel/preset-env',
```

Comment:
Would you mind adding an explanation why this configuration is needed?

Reply:
I still need to try if it works without this, in which case I'll of cause remove it.

Reply:
Not needed and is removed.